udf-compiler/src/test/scala/com/nvidia/spark/OpcodeSuite.scala

```diff
@@ -45,8 +47,8 @@ class OpcodeSuite extends FunSuite {
 
   import spark.implicits._
 
-// Utility Function for checking equivalency of Dataset type
-  def checkEquiv[T](ds1: Dataset[T], ds2: Dataset[T]) : Unit = {
+  // Utility Function for checking equivalency of Dataset type
+  def checkEquiv[T](ds1: Dataset[T], ds2: Dataset[T]): Unit = {
     ds1.explain(true)
     ds2.explain(true)
```

Comment:
I think we want to remove `.explain(true)` and `.show` as mentioned in https://github.com/NVIDIA/spark-rapids/pull/583

Reply:
Yes, removed. And for the rest diffs, I think I'll need to do a conflict resolve after another PR gets merged.

Reply:
Okay.  I have just added back "WIP".  We can remove it after the conflict resolution.